test/printing.jl

```diff
@@ -1,26 +1,28 @@
 @testset "Printing" begin
-  old_flag = Oscar.is_unicode_allowed()
+  if VERSION <= v"1.8.0"
```

Comment:
Perhaps add a comment along the lines of this:
```suggestion
  # TODO: disable this in Julia nightly for now until a Preferences.jl update is released
  # See https://github.com/oscar-system/Oscar.jl/issues/1140
  if VERSION <= v"1.8.0"
```